test/data/check-uk-visa-responses-and-expected-results.yml

```diff
@@ -695,7 +695,7 @@
   :responses:
   - kuwait
   - school
-  :next_node: :outcome_visit_waiver
+  :next_node: :outcome_school_waiver
```

Comment:
It looks as if this change was the result of this commit: 76b6d5cfaf66cabd4183d0bb4c9c0827880c4ccb which was part of #2524. I'm therefore happy to include it.